pontoon/base/templates/user.html

```diff
@@ -37,7 +37,7 @@
 {% block middle %}
 <ul class="info">
   <li><span class="fa fa-fw fa-envelope"></span><a href="mailto:{{ contributor.email }}">{{ contributor.email }}</a></li>
-  <li><span class="fa fa-fw fa-lock"></span>Can {% if contributor.has_perm('base.can_localize') %}submit and approve translations{% else %}make suggestions{% endif %}</li>
+  <li><span class="fa fa-fw fa-lock"></span>Can {% if contributor.can_approve_translations %}submit and approve translations{% else %}make suggestions{% endif %}</li>
```

Comment:
In a future PR we should consider listing the locales that the user has permission for. But this is great for now.